vector/src/main/res/layout/item_bottom_sheet_action.xml

```diff
@@ -82,4 +82,27 @@
         tools:ignore="MissingPrefix"
         tools:visibility="visible" />
 
+
+    <TextView
```

Comment:
would be great to get a screenshot of this view :pray:

Reply:
Will do!